apps/server/src/modules/board/service/column-board.service.spec.ts

```diff
@@ -152,6 +152,20 @@ describe('ColumnBoardService', () => {
 		expect(result).toEqual(copyStatus);
 	});
 
+	it('should copy Card', async () => {
+		const copyStatus: CopyStatus = { status: CopyStatusEnum.SUCCESS, type: CopyElementType.CARD };
+		columnBoardCopyService.copyCard.mockResolvedValueOnce(copyStatus);
+		const result = await service.copyCard({
+			originalCardId: '1',
+			sourceStorageLocationReference: { id: '1', type: StorageLocation.SCHOOL },
+			targetStorageLocationReference: { id: '1', type: StorageLocation.SCHOOL },
+			userId: '1',
+			targetSchoolId: new ObjectId().toHexString(),
+		});
+
+		expect(result).toEqual(copyStatus);
```

Comment:
```suggestion
		expect(returnedCopyStatus).toEqual(expectedCopyStatus);
```
just a suggestion. I think it makes it more understandable